### PR TITLE
PC-1229, PC-1228: Improve DataPagination

### PR DESCRIPTION
**Pull request by tomecko**

### CODE REVIEW

File: src/components/data-pagination/data-pagination.tsx [hunk 1]
```diff
@@ -47,15 +49,23 @@ class DataPaginationImpl<T> extends React.Component<IDataPaginationImplProps<T>>
...
+      onAsyncLimitChange, onLimitChange, style } = this.props;
     const { pagination, pendingUpdate, uuid } = context.state;
     return pagination.totalCount > 0 ? (
       <div className={cx(css.COMPONENT, 'panel-body', className)} style={style}>
         <DataPaginationSummary pagination={pagination} />
         <UpdateLoader pendingUpdate={pendingUpdate} />
         <DataPaginationDropdown
-          {...{ availableLimits, pagination, uuid }}
-          onLimitChange={Maybe.fromNull(onAsyncLimitChange).orJust(context.onLimitChange)}
+          {...{
```
Comment (dumpstate) [inline src/components/data-pagination/data-pagination.tsx]: I don't get the purpose of wrapping `...` within `{}`.

Comment (dumpstate) [reply src/components/data-pagination/data-pagination.tsx]: Why not just `{ availableLimits, ..., uuid }`?

Comment (tomecko) [reply src/components/data-pagination/data-pagination.tsx]: This is what JSX supports…

Comment (dumpstate) [reply src/components/data-pagination/data-pagination.tsx]: got it.

**Pull request merged**